### ISSUE TITLE
Records with values in `fu_fadmut` = 8 (Form A3) raise KeyError: '2a'

### ISSUE THREAD

**Issue by ldnicolasmay** [ctsit/nacculator#65]
I dug into this but can't quite figure it out. I get an error for the two records in our database that have `fu_fadmut` values of 8. Values of NA/null, 9 and 0 for `fu_fadmut` don't throw an error, only 8. (Other values for `fu_fadmut` are untested.) Maybe I'm barking up the wrong tree, but `fu_fadmut` values of 8 seems to be the only thing that's special or different about these 2 records.

Below is the output where fake IDs `UM11112561` and `UM11112562` result in a traceback that ends with `KeyError: '2a'`.

```
[START] ptid : UM11112561
[SKIP] Error for ptid : UM11112561
Traceback (most recent call last):
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/redcap2nacc.py", line 167, in convert
    warnings += check_blanks(packet)
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/redcap2nacc.py", line 38, in check_blanks
    if r(packet):
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/uds3/blanks.py", line 98, in should_be_blank
    return packet[key] == value
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/uds3/packet.py", line 39, in __getitem__
    raise KeyError(key)
KeyError: '2a'
[START] ptid : UM11112562
[SKIP] Error for ptid : UM11112562
Traceback (most recent call last):
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/redcap2nacc.py", line 167, in convert
    warnings += check_blanks(packet)
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/redcap2nacc.py", line 38, in check_blanks
    if r(packet):
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/uds3/blanks.py", line 98, in should_be_blank
    return packet[key] == value
  File "/Users/ldmay/Box Sync/Documents/nacculator/nacc/uds3/packet.py", line 39, in __getitem__
    raise KeyError(key)
KeyError: '2a'
```

Thanks.

**Comment by ufl-taeber** [comment]
@naomidb , did you fix this already?

**Comment by naomidb** [comment]
Yes I did. It looks like this was fixed with commit `75c0323a829982764cd1bc655abad2e247d0f48f`.
